reconciler/types.go

```diff
@@ -144,39 +143,6 @@ type Status struct {
 	id uint64
 }
 
-// statusJSON defines the JSON/YAML format for [Status]. Separate to
-// [Status] to allow custom unmarshalling that fills in [id].
```

Comment:
I still don't get why I did this thing with `id` and what I could've been thinking at the time. Right now I can't see any valid use-case for wanting to "fill in id", so let's just kill this.